.evergreen/spec.patch

```diff
@@ -27,15 +31,16 @@
  
  %build
  
-@@ -77,6 +75,7 @@
+@@ -77,6 +75,8 @@
  %cmake \
      -DCMAKE_BUILD_TYPE=Release \
      -DBSONCXX_POLY_USE_BOOST=1 \
 +    -DENABLE_UNINSTALL=OFF \
```

Comment:
Is removing the uninstall script also required to build successfully?

Reply:
Like the above line, this is also an unchanged context line.  That said, we did add this some time ago and it was probably needed for some obscure reason that escapes me at the moment.